research/picai/preprocessing_transforms.py

```diff
@@ -0,0 +1,176 @@
+from typing import Optional, Sequence, Tuple, Union
+
+import numpy as np
+import SimpleITK as sitk
+
+
+def resample_img(
+    image: sitk.Image,
+    spacing: Sequence[float] = (2.0, 2.0, 2.0),
+    size: Optional[Sequence[int]] = None,
+    is_label: bool = False,
+    pad_value: Optional[Union[float, int]] = 0.0,
+) -> sitk.Image:
+    """
+    Resample images to target resolution spacing.
+
+    Adapted from https://github.com/DIAGNijmegen/picai_prep.
+
+    Args:
+        image (sitk.Image): Image to be resized.
+        spacing (Sequence[float]): Spacing between voxels along each dimentsion.
+        size (Sequence[int]): Target size in voxels (z, y, x).
```

Comment:
lol...did they really order it as z,y,x?

Reply:
Yeah it hard to keep all the dimension conventions in your head hahaha Will definitely incorporate some tests to make sure were are getting dimensions right in follow up PR :)